drupal-8/README.md

```diff
@@ -0,0 +1,122 @@
+# Working with Drupal8 in cloud.gov (and Cloud Foundry)
+
+[Drupal](https://drupal.org) has been a popular PHP-based framework for content-management systems and web applications. It was not written as a cloud-native framework, so it takes a bit of tweaking to run on cloud.gov, or any other Cloud Foundry implementation. If you're on a _greenfield_ project you may want to consider other PHP frameworks, but if you're going forward with Drupal then this guide will get you there.
+
+This guide is written for [cloud.gov](https://cloud.gov/) users, but will work for any Cloud Foundry site. Just replace the specifics for `aws-rds` and `s3` with your site equivalents and everything should just work. 
+
+## Quickstart
+
+You can demonstrate a fully-functional Drupal install assuming you've already cloned this repo, and are using this directory:
+
+Update `manifest.yml` with:
+
+1. the correct admin name, `ACCOUNT_NAME`
+1. delete or change the `AWS_S3_ENDPOINT` if you're _not_ on cloud.gov
+
+Log in to Cloud Foundry (e.g. `cf login --sso -a https://api.fr.cloud.gov`), then run the following commands for cloud.gov:
+
+```
+cf create-service s3 basic-public d8ex-s3
+cf create-service aws-rds shared-mysql d8ex-db
+cf push d8ex --no-start -b  https://github.com/cloudfoundry/apt-buildpack.git
+# Set the ACCOUNT_PASS as an environment variable 
+cf set-env d8ex ACCOUNT_PASS "your-account-pass"
+cf v3-push d8ex -b https://github.com/cloudfoundry/apt-buildpack.git -b php_buildpack 
+```
+
+This project uses the "multi-buildpack" feature of Cloud Foundry, since we need `apt` to install the `mysql` client, and the `php_buildpack` for Drupal itself. The `v3-push` is experimental, so the syntax and usage may change.  Note: The two-push workflow is currently required because only `v3-push` supports multiple buildpacks.
+
+
+When the `v3-push` command completes:
+- Visit the site URL
+- Login with `ACCOUNT_NAME` and `ACCOUNT_PASS`
+- Set up use of S3 Flysystem instead of local disk:
+  - As a admin, go to Configuration -> File system, and set "Default download method" `Flysystem: s3`
+- On a default Drupal install, there should be two fields using the local filesystem. Those fields need to be updated to use Amazon S3:
+  - Image field on Article content type (Structure > Content types > Article > Manage fields > Image)
+  - Image field for User profile picture (Configuration > People > Account settings > Manage fields > Picture)
```

Comment:
These two Image field changes confused me because I clicked on the "Edit" button for those resources (which took me to the wrong tab) rather than clicking on the link that was next to the Edit button which took you to the proper tab).  But it was only a momentary confusion, so it's up to you if you want to hyper-clarify this.  Otherwise, this looks great to me!

Reply:
Yeah, that is confusing. I tried to fix the wording, but I'll just make that section go away on the next iteration by learning how Drupal can consume config.yml files to set these things in code.